Trainer.opy

```diff
@@ -2396,7 +2396,6 @@ def menu_hud():
         menu_off_opt_title[menu_opt_size] = "Slot 0 Hero Pool"
     menu_on_opt_title[menu_opt_size] = ""
     menu_on_opt_color[menu_opt_size] = default_menu_on_index_color
-    menu_hero_bar_color_set()

```

Comment:
Is this line necessary? If I remove it, it solves the error of setting the max kill count required to "RED".

Reply:
No it's not. It was just used to set the color, but it's superfluous.